main.go

```diff
@@ -49,16 +49,26 @@ func exec() int {
 
 	defer logFile.Close()
 
+	log := &logrus.Logger{
+		Out: os.Stdout,
+		Formatter: &logrus.TextFormatter{
+			ForceColors:      true,
+			DisableTimestamp: true,
+		},
+	}
+
+	log.SetLevel(logrus.InfoLevel)
+
 	h, err := os.Hostname()
 	if err != nil {
-		logrus.Debug(err)
+		log.Debug(err)
 
 		h = "unknown"
 	}
 
 	t := os.Getenv("FURYCTL_MIXPANEL_TOKEN")
 	if t == "" {
-		logrus.Debug("FURYCTL_MIXPANEL_TOKEN is not set")
+		log.Debug("FURYCTL_MIXPANEL_TOKEN is not set")
```

Comment:
after setting `log.SetLevel(logrus.InfoLevel)` we'll never see this debug message

Reply:
Should we rethink how we handle logrus logs in the cmd?

Reply:
mm maybe the whole logic is wrong, why are we doing that in the main.go?
```
	// Create the analytics tracker.
	a := analytics.NewTracker(t, versions[version], osArch, runtime.GOOS, "SIGHUP", h)
```

when we have in root.go the flag disable-analytics, shouldn't we wait for the flag to be parsed before creating a tracker?